### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by oriagranat9**

### CODE REVIEW

File: chameleon-server/src/main/java/com/chameleonvision/network/LinuxNetworking.java [hunk 1]
```diff
@@ -1,73 +1,102 @@
...
-	}
+    @Override
+    public boolean setDHCP() {
+        File dhcpConf = new File(PATH);
+        if (dhcpConf.exists()) {
+            try {
+                List<String> lines = FileUtils.readLines(dhcpConf, StandardCharsets.UTF_8);
+                for (int i = 0; i < lines.size(); i++) {
+                    String line = lines.get(i);
+                    if (line.contains("interface " + networkInterface.name)) {
```
Comment (Bankst) [inline chameleon-server/src/main/java/com/chameleonvision/network/LinuxNetworking.java]: Try checking using `startsWith(String)` to be sure we don't delete commented lines

File: chameleon-server/src/main/java/com/chameleonvision/vision/pipeline/pipes/SolvePNPPipe.java [hunk 2]
```diff
@@ -286,118 +308,69 @@ private MatOfPoint2f findBoundingBoxCorners(StandardCVPipeline.TrackedTarget tar
...
+    private MatOfPoint2f refineCornerEstimateSubPix(MatOfPoint2f corners, Mat greyImg) {
+        if(!shouldRefineCorners) return corners; // just return
+        Imgproc.cornerSubPix(greyImg, corners, winSize, zeroZone, criteria);
 
-        return goodFeaturesResultMat;
+        return corners;
     }
 
-    private StandardCVPipeline.TrackedTarget calculatePose(MatOfPoint2f imageCornerPoints, StandardCVPipeline.TrackedTarget target) {
+    NetworkTableEntry tvecE = NetworkTableInstance.getDefault().getTable("SmartDashboard").getEntry("tvec");
```
Comment (Bankst) [inline chameleon-server/src/main/java/com/chameleonvision/vision/pipeline/pipes/SolvePNPPipe.java]: Should this still be here? Maybe only enable it in debug mode? `--cv-development`

File: chameleon-server/src/main/java/com/chameleonvision/web/RequestHandler.java [hunk 3]
```diff
@@ -53,8 +54,17 @@ public static void onGeneralSettings(Context ctx) {
...
             ConfigManager.settings.hostname = (String) map.get("hostname");
             ConfigManager.saveGeneralSettings();
+            // setting up network config after saving
+            boolean isStatic = ConfigManager.settings.connectionType.equals(NetworkIPMode.STATIC);
+
+            if (NetworkManager.setHostname(ConfigManager.settings.hostname) &&
+                    NetworkManager.setNetwork(isStatic, ConfigManager.settings.ip, ConfigManager.settings.netmask, ConfigManager.settings.gateway)) {
+                ctx.status(200);
+            } else {
+                ctx.result("something went wrong while setting network configuration");
```
Comment (Bankst) [inline chameleon-server/src/main/java/com/chameleonvision/web/RequestHandler.java]: Capitalization

File: chameleon-server/src/main/java/com/chameleonvision/vision/pipeline/pipes/SolvePNPPipe.java (hunk 2 above)
Comment (oriagranat9) [reply chameleon-server/src/main/java/com/chameleonvision/vision/pipeline/pipes/SolvePNPPipe.java]: commented it for now if its important @mcm001 can add it back

**Pull request merged**